changelog-preview.sh

```diff
@@ -7,7 +7,20 @@ function _changelogsh_preview {
     raw_version=$1
   fi
 
-  if [ $raw_version != "Unreleased" ]; then
+  if [ $raw_version == "-all" ]; then
```

Comment:
```suggestion
  if [ $raw_version == "--all" ]; then
```
single dash is normally for one letter shortcuts like `-r` for `--recursive` and `-d` for `--daemon`. double dash is more appropriate here for `--all`

Reply:
Thanks for the review @clrke. Awesome as always!

However, I suggest we keep "-all" like how "find" accepts options since this tool won't be a full blown cli tool and won't be having any other options soon (even short and long version of "all")

But I defer to what @xdgimf @TheoryOfNekomata might think.